frontend/src/admin/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {useState} from "react";
+import DropdownMenuIcon from "../dropdown-menu-icon/dropdown-menu-icon";
+import DropdownMenuItem from "../dropdown-menu-item/dropdown-menu-item";
+import "./dropdown-menu.scss";
+
+export default function DropdownMenu() {
+    const [showDropdown, setShowDropdown] = useState<boolean>(false);
+
+    const toggleDropdown = () => {
+        setShowDropdown(!showDropdown);
+    };
+
+    const hideDropdown = (event: React.FocusEvent<HTMLButtonElement>): void => {
+        if (event.currentTarget === event.target) {
+            setShowDropdown(false);
+        }
+    };
+
+    return (
+        <div className={showDropdown ?"c-dropdown-menu--active" : "c-dropdown-menu"}>
+            <button
+                onClick={() => toggleDropdown()}
+                onBlur={(e: React.FocusEvent<HTMLButtonElement>): void => hideDropdown(e)}
+            >
+                <DropdownMenuIcon/>
+            </button>
+            <div className="c-dropdown-menu-items">
+                <DropdownMenuItem id="download" name="Download user"/>
```

Comment:
This component still has the generic name `DropdownMenu`, but continues to specify actions that can be used on the `User` entity

Reply:
✅ This is changed  - i have implemented the children props